packages/base/src/formbuilder/formdialog.tsx

```diff
@@ -53,54 +49,18 @@ export class FormDialog extends Dialog<IDict> {
     const jgisModel = options.model;
     const body = (
       <div style={{ overflow: 'hidden' }}>
-        <ObjectPropertiesForm
-          parentType="dialog"
-          filePath={`${filePath}::dialog`}
+        <BaseForm
```

Comment:
Assuming you resolve the other comment and rename this class `ProcessingFormDialog`.

Now you need some logic here to pick the right form depending on the processing task you want to achieve. 

So it won't be `BaseForm` here, you need to select `BaseForm` only if we don't have a custom form defined (e.g. for buffer). For dissolve, you will need to use your new `DissolveForm` class.

See how we do things with the `CreationFormDialog` for layers, where we pick the right form with https://github.com/geojupyter/jupytergis/blob/main/packages/base/src/formbuilder/formselectors.ts

You will need a selector too, picking the right processing form depending on the processing type.